pkg/e2e/harness_runner/harness_runner.go

```diff
@@ -29,8 +29,11 @@ var (
 
 var _ = ginkgo.Describe("Test harness", ginkgo.Ordered, ginkgo.ContinueOnFailure, label.TestHarness, func() {
 	harnesses := viper.GetStringSlice(config.Tests.TestHarnesses)
-	timeoutInSeconds = 3600 * viper.GetInt(config.Tests.SuiteTimeout)
-
+	if viper.IsSet(config.Tests.HarnessTimeout) {
```

Comment:
I am okay with the new setting for a harness timeout making it specific/easy to associate to the other test harness settings. I do think we should have a follow up action to deprecate the `POLLING_TIMEOUT` setting. Just to reduce any confusion on which setting to use. It could be taken in a phased approach:

Phase 1: Remove using `POLLING_TIMEOUT` for test harness runner and remove this setting from all test harness jobs. Removal means PR's would need to be opened to adjust them to use the new setting `HARNESS_TIMEOUT`. We could even do it in this PR (https://github.com/openshift/release/pull/41828) I have opened.

Phase 2: Review osde2e and drop support for `POLLING_TIMEOUT` anywhere else its required. I think most of its use is passed to the ginkgo it nodes for timeouts but I believe those are not valid anymore now that we directly call `ginkgo.It()` over using the wrapper we used to have in osde2e. Of course this one would be a `low` priority. :)

This way osde2e has 2 primary timeout settings:

1. `SUITE_TIMEOUT` controls the overall duration ginkgo will run the test suite.
2. `HARNESS_TIMEOUT` controls the timeout to wait for a test harness pod to be finished.

These are just some of my thoughts, I am interested to hear your thoughts/the rest of the team!

Reply:
Sounds good @ritmun